.github/workflows/ci.yml

```diff
@@ -17,7 +17,6 @@ on:
       - 'requirements/dev.txt'
       - '.docker.env'
       - '**/.gitignore'
-      - '.python-version'
```

Comment:
This doesn't happen often enough to ignore it.

Reply:
:+1: Actually I think we _do_ want to run CI on a PR that just changes `.python-version`, so I don't think this should have been here.

`ci.yml` comes from the cookiecutter so you'll have to backport this change to the cookiecutters repo